src/models/tiles/geometry/geometry-migrate.ts

```diff
@@ -31,14 +31,19 @@ export const convertChangesToModel = (changes: JXGChange[]) => {
   return exportGeometryModel(changesJson);
 };
 
-export const convertModelToChanges = (model: GeometryBaseContentModelType, addBuffers?: boolean): JXGChange[] => {
+export const convertModelToChanges = (
+  model: GeometryBaseContentModelType, addBuffers?: boolean, includeUnits?: boolean
+): JXGChange[] => {
```

Comment:
Rather than multiple boolean params, I'd use an options object:
```suggestion
interface IConvertOptions {
  addBuffers?: boolean;
  includeUnits?: boolean;
}
export const convertModelToChanges = (model: GeometryBaseContentModelType, options?: IConvertOptions): JXGChange[] => {
  const { addBuffers = false, includeUnits = false } = options || {}
```